src/sass/common/reality-check.scss

```diff
@@ -21,7 +21,7 @@
     background-color: #fff;
     border-radius:    .3em;
 }
-#reality-check > div > div > div:first-child {
+.lightbox > div > div > div:first-child {
```

Comment:
perhaps we now want to identify the elements inside the .lightbox > div > div box differently. lightbox is a class and hence can contain various things.

Maybe it's a good idea to give the parent of this box separate ids for both, reality-check and this balance box.